src/migrations/versions/85ed22b90d72_add_queuedfile.py

```diff
@@ -0,0 +1,35 @@
+"""add queuedfile
+Revision ID: 85ed22b90d72
+Revises: 6b495d5a4855
+Create Date: 2024-10-27 18:35:05.991934
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel
+
+
+# revision identifiers, used by Alembic.
+revision = "85ed22b90d72"
+down_revision = "6b495d5a4855"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "queuedfile",
```

Comment:
Should we add something like "indexed" column so we know which files are already indexed?
It would easier the workflow and allow to delete indexed files easily.

Reply:
not "indexed", but other files will be added. Indexed is not necessary, because indexed files will be removed from queue.